it/uk/gov/hmrc/agentrelationships/stubs/DesStubs.scala

```diff
@@ -151,7 +151,7 @@ trait DesStubs {
           .withBody(s"""{"reason": "Service unavailable"}""")))
   }
 
-  def getClientActiveAgentRelationships(encodedClientId: String, service: String, agentArn: String): Unit = {
+  def getClientActiveAgentRelationshipsItSa(encodedClientId: String, service: String, agentArn: String): Unit = {
```

Comment:
You could add service=ITSA in the below url, instead of passing as a param since its service has been mentioned in the method name

Reply:
:+1: